### DISCUSSION TITLE
Insert /announcecolor in the options

### DISCUSSION THREAD

**Discussion by Pafon22** [Chatterino/chatterino2#4367] [Ideas] [answered]
Unrecognized command when sending /announce with any color.
Insert /announceblue /announcegreen /announceorange /announcepurple in the options when possible. I don't know if Twitch already has the endpoint for this.

**Comment by iProdigy** [comment, marked as answer]
Currently only `/announce` is supported which uses the primary color: https://github.com/Chatterino/chatterino2/blob/d38187f794f0c49ba0f3926b01be7de217a48fa9/src/controllers/commands/CommandController.cpp#L2028

The color versions exist in Helix, just no PR has been created yet to add the chat commands to the controller: https://github.com/Chatterino/chatterino2/pull/4003#issuecomment-1256961055

**Reply by Mm2PL** [reply]
It's worth noting that we need to implement displaying colored announcements first

**Comment by Mm2PL** [comment]
Accepted as issue #5218.
